src/interfaces/task.ts

```diff
@@ -0,0 +1,16 @@
+export default interface test {
+    id?: number,
+    title: string,
+    description: string,
+    created_at: string,
+    due_date: string,
+    project: string,
+    priority: string,
+    color: string,
+    section: string,
+// eslint-disable-next-line semi
```

Comment:
remove this line

Reply:
I need it so I can commit , thanks to husky and eslint